src/lib/controller.ts

```diff
@@ -145,6 +160,170 @@ class FastAuthController {
   async signTransaction(params) {
     return this.signDelegateAction(params);
   }
+
+  // This call need to be called after new oidc token is generated
+  // https://github.com/near/mpc-recovery#claim-oidc-id-token-ownership
+  async claimOidcToken(oidcToken) {
+    const CLAIM_SALT = 3177899144 + 0;
+    const keypair = await this.getKey();
+    const signObj = getUserCredentialsFrpSignature({
+      salt: CLAIM_SALT,
+      oidcToken,
+      shouldHashToken: true,
+      keypair,
+    });
+
+    const data = {
```

Comment:
It should be a good idea to introduce types for requests.

Reply:
in JS, it is basically all parsed to string at the moment..